Localization/en-US/Mods.PathOfTerraria.hjson

```diff
@@ -4,9 +4,9 @@ Keybinds: {
 	UseSkill3.DisplayName: Use Skill3
 	UseSkill4.DisplayName: Use Skill4
 	UseSkill5.DisplayName: Use Skill5
-	WeaponSwap.DisplayName: Swap Weapons
 	CopyItemInfo.DisplayName: Copy Item Info
 	StatUIKey.DisplayName: Toggle Stat Page
+	GearSwap.DisplayName: Swap Gear
```

Comment:
I think I like `Swap Weapons` vs `Swap Gear` more for the keybind as it should only be swapping the player's mainhand and offhand.

Gear swap makes me think ur swapping the entire gearset to one of the 3 gearset choices.

Reply:
That's reasonable, but I felt like calling it weapon swap was innacurate since you're also changing the offhand *accessory*. Perhaps "Swap Hands"?

Reply:
Swap Hands definitely makes more sense, yeah.